backend/DataAccess/Configuration/LfIdSerializer.cs

```diff
@@ -149,3 +197,19 @@ public void Serialize(BsonSerializationContext context, BsonSerializationArgs ar
         public Type ValueType => _lfIdSerializer.ValueType;
     }
 }
+
+
+/// <summary>
+/// Ensures that our LfIdSerializerProvider is used for all LfId properties including dictionaries
+/// LfIdSerializerProvider.GetSerializer is not called for dictionary members, so we apply it here after each member has been mapped
+/// </summary>
+public class EnsureLfIdMemberSerializationConvention : ConventionBase, IMemberMapConvention
```

Comment:
No, `CanConvert` isn't for serializing for Mongo, it's for generating JSON for the web API.

Reply:
oh yeah that makes sense.

So I'm not sure if I'm missing something but I just ran a unit test using the debugger and put a breakpoint in `GetSerializer` and it gets called for our dictionary type, so I don't think we need this since it already works. Just try commenting out the code in the `Apply` method.

Reply:
You're right 😭 ... I mean, that's good, but some unfortunate combination of events over a prolonged period of time made me think that this wasn't the case...man of man. .... ☕🏃